imadityayadav/frontend
# Implemented Dark Mode Toggle Button

Summary
This pull request introduces a dark mode feature to enhance user experience by providing an alternative color scheme that is easier on the eyes, especially in low-light conditions.

##Changes Made
Toggle Switch: Added a toggle switch in the navbar that allows users to switch between light and dark modes.
Color Scheme Updates: Updated the styles across components to reflect the new dark color scheme, ensuring text readability and visual consistency.
Accessibility Enhancements: Ensured that all text remains readable and all interactive elements are accessible in dark mode.

##How to Test
Use the toggle switch to change between light and dark modes.
Verify that all UI elements update correctly according to the selected mode.

##Screenshots
--Before 
![image](https://github.com/user-attachments/assets/cd36cccd-7523-469a-93f2-cd17d34d256d)
![image](https://github.com/user-attachments/assets/3d73ad54-c847-407b-8634-b75870ee9864)

--After
![image](https://github.com/user-attachments/assets/173b0d6e-3969-4005-aefb-89ccb25df729)
![image](https://github.com/user-attachments/assets/27b75d7e-8599-4eb4-b740-76b8d00f1b61)

Closes #28
